src/converters/degiroConverterV3.ts

```diff
@@ -27,10 +27,14 @@ export class DeGiroConverterV3 extends AbstractConverter {
       delimiter: ",",
       fromLine: 2,
       columns: this.processHeaders(input),
-      cast: (columnValue, _) => {
+      cast: (columnValue, context) => {
 
         // Custom mapping below.
 
+        if (context.column === "currency" && columnValue === "GBP") {
```

Comment:
This is a wrong move.

Consider this record:
  03-03-2020,11:14,03-03-2020,ISHARES GLOBAL CLEAN ENERGY UCITS ETF,IE00B1XNHC34,"Koop 475 @ 583,5 GBX",,GBP,-2771.63,GBP,,3b000105-xxxx-xxxx-xxxx-xxxxxxxxxxxx

The record represents purchasing 475 stocks each at 583 GBX (GBp, pennies). However, the total cost of 2771.63 is correct and in GBP (pounds): 475*5.835=2771.63. Converting GBP to GBp here would lead to the total cost being be 100x smaller.

Reply:
I’m going to take another look at it tomorrow. Had not had a change to be at a PC today.

The original change came from this line, which is similar to what you said above:

```cab
02-04-2024,09:00,02-04-2024,AVIVA,GB00BPQY8M80,Sell 4 AVIVA@496 GBX (GB00BPQY8M80),,GBP,19.84,GBP,114.31,86c1f17b-8a74-4126-af39-61049bcb6e33
```

I think I mixed it up with GBX, which is the same as GBp.